Code/Mantid/Framework/API/test/ExperimentInfoTest.h

```diff
@@ -676,6 +685,25 @@ class ExperimentInfoTest : public CxxTest::TestSuite
 
   }
 
+  void test_nexus_intrument_info()
+  {
+     ExperimentInfo ei;
+
+    // We get an instrument group from a test file in the form that would occur in an ISIS Nexus file
+
+    // Create the root Nexus class
+    std::string path = FileFinder::Instance().getFullPath("LOQinstrument.h5");
+
+   
+    // Get nexus file for this and move it to ISIS Nexus group 'raw_data_1'
+    ::NeXus::File nxFile(path, NXACC_READ);
+
+    // Load the Nexus IDF info
+    std::string params;
+    TS_ASSERT_THROWS_NOTHING(ei.loadInstrumentInfoNexus( &nxFile, params));
+
+  }
```

Comment:
This is a good start to a test but can we not also check some things about the `ei` object after loading? It's good that we check it doesn't throw but we need to also verify it has actually done the correct thing.